### ISSUE TITLE
xNN New Features - UI/UX discussion / suggestions

### ISSUE THREAD

**Issue by Thorlogy** [OpenRoberta/openroberta-lab#1584]
1. Change the color of the Frame (input training data set) from black to turquoise - like the frame of the sensor data view.
2. Possibility to change the size of the window for entering training data (enlarge / reduce).
3. Possibility to set random numbers only for weights (and not for biases)

**Comment by Thorlogy** [comment]
4. If you are in the "Neuronal Networks Define" tab, the pop-up training data window should remain open as you enter the input data layer by layer. 
5. It could be very helpful to show the formulas (or the methodes?) used for the calculation of the training loss and for the calculation of the edge weights + bias?

**Comment by Thorlogy** [comment]
6. It could be very helpful if you could easily record sensor data and then use it as training data. This could be done via an extra block in the NEPO programme, for example. You could make settings here, 
- which sensor
- Data rate (i.e. every second) 
- Type of data (raw data, ...)

**Comment by Thorlogy** [comment]
7. Labelling the axes - for the training lock graphic
